CHANGELOG.md

```diff
@@ -7,3 +7,17 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## Added
 - Basic Skel
 - missing PR template
+
+## [0.0.2] - 2016-08-11
```

Comment:
Opened up #2 to figure out what to do with the version on rubygems.org the version won't be an issue but pushing to rubygems in general will be until we figure something out.